app/controllers/BaseController.scala

```diff
@@ -30,10 +31,15 @@ abstract class BaseController(implicit val appConfig: AppConfig) extends Fronten
   implicit val ec: ExecutionContext
 
   def technicalDifficulties(implicit request: Request[_], messages: Messages)  = {
-
     implicit val loggedInUser = LoggedInUser(None)
 
     InternalServerError(error_template("Technical difficulties", "Technical difficulties",
       "Sorry, we’re experiencing technical difficulties"))
   }
-}
+
+  def notFound(errors: String)(implicit request: Request[_], messages: Messages) : Future[Result] = {
```

Comment:
Why does this function return a Future whilst the technicalDifficulties one above doesn't?
They both seem to be helpers for routing to other pages but work differently.
It feels like they should both be working in the same way.